cinema/serializers.py

```diff
@@ -125,9 +125,9 @@ class TicketSerializer(serializers.ModelSerializer):
     def validate(self, attrs):
         data = super(TicketSerializer, self).validate(attrs=attrs)
         Ticket.validate_ticket(
-            attrs["row"], 
-            attrs["seat"], 
-            attrs["movie_session"].cinema_hall, 
+            attrs["row"],
+            attrs["seat"],
+            attrs["movie_session"].cinema_hall,
             ValidationError
```

Comment:
In the `validate` method of `TicketSerializer`, the `validate_ticket` method is called with `ValidationError` as an argument. Ensure that `validate_ticket` is designed to accept `ValidationError` as a parameter. If not, you may need to handle the validation error differently.